src/backend/expungeservice/expunger/analyzers/time_analyzer.py

```diff
@@ -71,6 +72,22 @@ def _mark_as_time_ineligible(charges, reason, eligibility_date):
         for charge in charges:
             charge.set_time_ineligible(reason, eligibility_date)
 
+    @staticmethod
+    def _set_arrests_as_eligible(expunger):
+        # If most recent conviction is time eligible
+        # AND all other charges are arrests
+        # Then make all other charges time eligible
+        if expunger.most_recent_conviction.expungement_result.time_eligibility.status:
+            arrest_dispositions = {'Dismissed', 'Acquitted', 'Dismissal', 'No Complaint'}
+            mrc = expunger.most_recent_conviction
+            non_mrc_charges = [x for x in expunger.charges if x != mrc]
```

Comment:
We're only concerned with charges in the same case as the mrc; all other charges are unaffected.

Reply:
@NickSchimek Cool, I wasn't sure if the expunger processed more than one case at a time - wanted to follow up on that.

Reply:
ask questions anytime. A charges time-eligibility is determined by all the charges in the record. Hence, the list of charges is every charge in the record.

Reply:
Ah, makes perfect sense, thank you